pathd/path_pcep_controller.c

```diff
@@ -400,11 +428,12 @@ int pcep_thread_get_counters_callback(struct thread *t)
 	assert(ctrl_state != NULL);
 	struct pcc_state *pcc_state;
 
-	if (args->pcc_id <= ctrl_state->pcc_count) {
 		pcc_state = get_pcc_state(ctrl_state, args->pcc_id);
```

Comment:
Indentation got wrong ?

Reply:
changed

Reply:
The indentation still looks wrong, at least here in the github review interface.